osf/models/preprint_provider.py

```diff
@@ -40,13 +40,27 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     allow_submissions = models.BooleanField(default=True)
     additional_providers = fields.ArrayField(models.CharField(max_length=200), default=list, blank=True)
 
-    PREPRINT_WORD_CHOICES = (
-        ('preprint', 'Preprint'),
-        ('paper', 'Paper'),
-        ('thesis', 'Thesis'),
-        ('none', 'None')
+    PREPRINT = 'preprint'
+    PREPRINTS = 'preprints'
+    PAPER = 'paper'
+    PAPERS = 'papers'
+    THESIS = 'thesis'
+    THESES = 'theses'
+
+    PREPRINT_WORD_CHOICES_SINGULAR = (
```

Comment:
This was not what I was hoping for. One of the big parts of the request was that we didn't want multiple lists to maintain, and now we have three instead of one.

How about instead:
```py
PREPRINT_WORD_CHOICS = [
    ['preprint', 'Preprint', 'Preprints'],
    ['thesis', 'Thesis', 'Theses'],
   (and so on)
]

preprint_singulars = tuple iteration of items 0 and 1 from each item in PREPRINT_WORD_CHOICES
preprint_word = models.CharField(max_length=10, choices=preprint_singulars, default='preprint')

@property
preprint_word_plural():
    Here just look up the current preprint word and return the plural version
```

Reply:
Okay.